cms/static/sass/views/_video-upload.scss

```diff
@@ -17,6 +17,29 @@
     top: 0 !important;
   }
 
+  .button-link {
+    background:none;
+    border:none;
+    padding:0;
+    color: $ui-link-color;
+    cursor:pointer
+  }
+
+  .show-video-transcripts-wrapper {
+    display: block;
+  }
+  .show-video-transcripts-wrapper.hidden {
+    display: none;
+  }
+
+  .show-video-transcript-content {
+    margin-top: 20px;
+    .transcript-language-menu {
+      display: block;
+      width: 200px;
```

Comment:
should use $baseline calculation for `margin-top` and `width` above.

Reply:
I don't think baseline would be suitable for width, however I am using for margin